scripts/split-e2e-tests.ts

```diff
@@ -296,14 +296,16 @@ const splitTests = (
         ...JSON.parse(JSON.stringify(baseJobLinux)), // deep clone base job
         identifier: getIdentifier(names),
       };
-      tmp.env.variables = {};
+      tmp.env.variables = tmp.env.variables ?? {};
       tmp.env.variables.TEST_SUITE = j.tests.join('|');
       tmp.env.variables.CLI_REGION = j.region;
       if (j.useParentAccount) {
         tmp.env.variables.USE_PARENT_ACCOUNT = 1;
       }
       if (j.runSolo) {
-        tmp.env['compute-type'] = 'BUILD_GENERAL1_SMALL';
+        tmp.env['compute-type'] = 'BUILD_GENERAL1_MEDIUM';
+        // BUILD_GENERAL1_SMALL has 7GB of memory. 6656 = 6.5GB. Leave 0.5GB for the OS and other processes.
```

Comment:
Is this comment relevant since we're using MEDIUM? Or does the comment incorrectly specify SMALL when it should be specifying MEDIUM?

Reply:
Should be medium. I'll update.